README.md

```diff
@@ -6,26 +6,81 @@ Implements https://groups.google.com/forum/#!msg/kubernetes-sig-api-machinery/rH
 
 ## Purpose
 
-This controller implements the `EtcdStorage` type, used to provide etcd storage for aggregated API servers.
+This controller implements the `EtcdStorage` type, used to provide etcd storage for the aggregated API servers.
 
 ## Compatibility
 
 HEAD of this repo matches versions `1.10` of `k8s.io/apiserver`, `k8s.io/apimachinery`, and `k8s.io/client-go`.
 
-## Running
+## Running controller
 
-Prerequisite: Since the sample-controller uses apps/v1 deployments, the Kubernetes cluster version should be greater than 1.9.
+There're two ways to run `EtcdProxyController`:
 
+* in-cluster, which is done by deploying [the deployment manifest, located in the `artifacts/deployment` directory](https://github.com/xmudrii/etcdproxy-controller/blob/master/artifacts/deployment/00-etcdproxy-controller.yaml),
```

Comment:
does a relative path work? This might break and point to a wrong branch/tag.

Reply:
Updated all URLs to use relative paths.